pysoa/server/logging.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import, unicode_literals
+
+import logging
+import threading
+
+
+class PySOALogContextFilter(logging.Filter):
+    def __init__(self):
+        super(PySOALogContextFilter, self).__init__('')
+
+    def filter(self, record):
+        context = self.get_logging_request_context()
+        if context:
+            record.correlation_id = context.get('correlation_id') or ''
```

Comment:
Not that I disagree in any way with this, but I had thought that correlation_id was an ebsoa thing rather than a pysoa thing, although I would argue that it ought to be a base level pysoa thing anyway.

Reply:
It's not just n EB SOA thing. It's base PySOA: https://github.com/eventbrite/pysoa/blob/73e8746f7aafd969b0649b9175daf2af77b59f7d/pysoa/client/client.py#L402-L424